### PR TITLE
Silvestreperret/add cs runner

### PR DESCRIPTION
**Pull request by SilvestrePerret**

### CODE REVIEW

File: .github/workflows/main.yml
Comment (badouralix) [inline .github/workflows/main.yml:98]:
```suggestion
```

File: tool/runners/cs.py
Comment (badouralix) [inline tool/runners/cs.py:13]:
```suggestion
        # Create a temporary directory to put the compiled C# in,
```

File: tool/runners/cs.py
Comment (badouralix) [inline tool/runners/cs.py:51]: Trying to fix the no newline at end of file ( not sure it works, maybe you need to make the change locally )

```suggestion
        return SubmissionCs(self.file)
```

File: tool/templates/template.cs
Comment (badouralix) [inline tool/templates/template.cs:24]: Trying to fix the no newline at end of file ( not sure it works, maybe you need to make the change locally )

```suggestion
}
```

**Review by badouralix — APPROVED** [review]
Added a few nitpicks

As long as it does roughly what we need, lgtm :shipit:
And we can still fix things in the future if ever we discover a new pitfall

**Comment by SilvestrePerret** [comment]
@badouralix 
I will wait until some submissions are available to make sure everything run smoothly on the CI.
I will rebase my submission branches on this one until it's merged.

**Pull request merged**
